run_rayci.sh

```diff
@@ -8,7 +8,7 @@ TMP_DIR="$(mktemp -d)"
 
 echo "--- Install rayci"
 
-curl -sL 'https://go.dev/dl/go1.20.6.linux-amd64.tar.gz' | tar -xzf - -C "$TMP_DIR"
+curl -sfL 'https://go.dev/dl/go1.21.4.linux-amd64.tar.gz' | tar -xzf - -C "$TMP_DIR"
```

Comment:
where is this script used? does it need to support arm as well?

Reply:
rayci and this script runs only on the very first step, for bootstrapping, so this does not need to support arm64 (or at least not important).

wanda needs to support arm64.